src/System.Text.Primitives/System/Text/Encoding/Utf8/Utf16LittleEndianStringExtensions.cs

```diff
@@ -1,15 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-namespace System.Text.Utf8
-{
-    internal static class Utf16LittleEndianStringExtensions
-    {
-        // TODO: Naming it Equals causes picking up wrong overload when compiling (Equals(object))
-        public static bool EqualsUtf8String(this string left, Utf8String right)
```

Comment:
I removed it because it is not being used at all. Given it was internal only, nobody external could have accessed it either.